test/DynamoCoreWpfTests/NodeAutoCompleteSearchTests.cs

```diff
@@ -164,35 +164,82 @@ public void NodeSuggestions_InputPortBuiltInNode_AreCorrect()
             Assert.AreEqual("string", type);
 
             var searchViewModel = (ViewModel.CurrentSpaceViewModel.NodeAutoCompleteSearchViewModel as NodeAutoCompleteSearchViewModel);
-            searchViewModel.PortViewModel = inPorts[1];
+            searchViewModel.PortViewModel = inPorts[0];
             var suggestions = searchViewModel.GetMatchingSearchElements();
-            Assert.AreEqual(22, suggestions.Count());
+            Assert.AreEqual(70, suggestions.Count());
```

Comment:
Or this one?

Reply:
there are now 70 suggestions for `var[]..[]`

Reply:
Well, as per my comment above, it should be skipped.

Reply:
in this case, do you want me to include another node, that does show some suggestions and then assert it, or asserting 0 suggestions would suffice?